filebeat/fileset/pipelines.go

```diff
@@ -161,23 +162,77 @@ func setECSProcessors(esVersion common.Version, pipelineID string, content map[s
 		return fmt.Errorf("'processors' in pipeline '%s' expected to be a list, found %T", pipelineID, p)
 	}
 
-	minUserAgentVersion := common.MustNewVersion("6.7.0")
-	for _, p := range processors {
+	// A list of all processor names and versions to be checked.
+	versionChecks := Processors{
```

Comment:
I'd like to have this data moved out of the function and declare it in the package. Like

`var processorCompatibilityChecks = []processorCompatibility{...`